### PR TITLE
Added instructions for setting up and running the examples within the re...

### PR DESCRIPTION
**Pull request by jondot**
...pository

### CODE REVIEW

**Comment by jondot** [comment]
Just signed the CLA.

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -29,3 +29,13 @@ In the newly created folder `AwesomeProject/`
 
 Congratulations! You've just successfully run and modified your first React Native app.
 
+## Running the Examples
+
```
Comment (amasad) [inline docs/GettingStarted.md]: remove space

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -29,3 +29,13 @@ In the newly created folder `AwesomeProject/`
...
+
+- `git clone git@github.com:facebook/react-native.git`
+- `cd react-native && npm install`
+- `cd Examples`
+
+Now open any example and hit run in Xcode.
+
+
```
Comment (amasad) [inline docs/GettingStarted.md]: remove space

**Comment by amasad** [comment]
good idea. Let's just remove extra spaces and happy to pull it

**Comment by jondot** [comment]
This good?

**Comment by amasad** [comment]
Oh sorry, I thought this was an addition to this repo's readme. I don't think we should add more stuff to the getting started tutorial. It should be as small as possible.

However, I think this would be a great addition to this repo's readme. https://github.com/facebook/react-native/blob/master/README.md

**Comment by jondot** [comment]
Sure, I had the dilemma myself since I didn't see any concrete 'getting started' type wording there, I figured it belonged in the `/docs` folder. I'll add it as a separate pull request and then you can just close this one and accept the other one :)

**Comment by amasad** [comment]
Fixed via #267

**Pull request closed without merging**